packages/messages/src/browser/style/notifications.css

```diff
@@ -15,188 +15,188 @@
  ********************************************************************************/
 
 .theia-NotificationsContainer {
-    pointer-events: none;
```

Comment:
@BroKun Could you elaborate please what really was changed? It would be good to separate a refactoring change in a separate commit to simplify a review. We are going to include this rule in the review guideline: https://github.com/theia-ide/theia/pull/5616#issuecomment-508428861